packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.test.tsx

```diff
@@ -87,87 +83,100 @@ describe('components/PaymentConsentCheckbox', () => {
         expect(legalCheckbox.props.vars.intervalCount).toBe(
           plan.interval_count
         );
-        expect(legalCheckbox.props.children.props.children).toBe(expectedMsg);
+
+        expect(legalCheckbox.props.children.props.children[0]).toBe(
+          'I authorize Mozilla, maker of Firefox products, to charge my payment method '
+        );
+        expect(
+          legalCheckbox.props.children.props.children[1].props.children
+        ).toMatch(
+          /\$\d+[.]\d{2}[ ][daily|every 6 days|weekly|every 6 weeks|monthly|every 6 months|yearly|every 6 years]/
```

Comment:
This would work, however it runs the risk of returning a false positive. (If for some reason, as a bug, `formatPlanPricing` returns "daily" regardless of input, all these tests will still pass, even though there's a bug)

This shouldn't hold up the change though, so feel free to squash and merge.